mne/stats/tests/test_cluster_level.py

```diff
@@ -275,10 +268,7 @@ def test_cluster_permutation_t_test(numba_conditional, stat_fun):
 def test_cluster_permutation_with_connectivity(numba_conditional):
     """Test cluster level permutations with connectivity matrix."""
     try:
-        try:
-            from sklearn.feature_extraction.image import grid_to_graph
-        except ImportError:
-            from scikits.learn.feature_extraction.image import grid_to_graph
+        from sklearn.feature_extraction.image import grid_to_graph
```

Comment:
This `try/except` is old and ugly. Better to just directly do:
```
    from sklearn.feature_extraction.image import grid_to_graph
```
and decorate each function that needs it with
```
@requires_sklearn
```

Reply:
I wondered about thata. fixed now.